src/composables/types/common/evidence.ts

```diff
@@ -0,0 +1,12 @@
+import type { ChoiceIdType } from '@/composables/types';
+
+export type EvidenceType = {
+  id: string,
+  choice_id: ChoiceIdType,
```

Comment:
Could we use `camelCase` in the key-names? It's JS-preferrable syntax and we could use `humps` to `decamelize` and `camelize` DTO before/after request.

Reply:
Yes, I add humps to our project